docs/iris/src/whatsnew/contributions_2.0/incompatiblechange_2017-Oct-18_remove-exp-ff.txt

```diff
@@ -1 +1,8 @@
-* Removed deprecated module ```iris.experimental.fieldsfile```.
\ No newline at end of file
+* Removed deprecated module ```iris.experimental.fieldsfile```.
+  Note that there is no direct replacment for
+  ```:meth:iris.experimental.fieldsfile.load```, which specifically performed
+  fast loading from _either_ PP or FF files.
+  Instead, please enable `:meth:iris.fileformats.um.structured_um_loading` and
```

Comment:
This does not make it clear what the replacement is, it needs to be explained in more detail, an example may be useful.

Reply:
I guess this will cross reference the docs at http://scitools.org.uk/iris/docs/latest/iris/iris/fileformats/um.html#iris.fileformats.um.structured_um_loading

Reply:
> this will cross reference the docs at ... iris.fileformats.um.structured_um_loading

Yes, I thought that would be good enough : as it's only a whatsnew item, I think it can be a pointer rather than full info, no ?